src/EnergyPlus/SurfaceGeometry.cc

```diff
@@ -112,6 +112,10 @@ namespace SurfaceGeometry {
 
     static std::string const BlankString;
 
+    int constexpr UnenteredAdjacentZoneSurface = -998; // allows users to enter one zone surface ("Zone")
+                                                       // referencing another in adjacent zone
+    int constexpr UnreconciledZoneSurface = -999;      // interim value between entering surfaces ("Surface") and reconciling
```

Comment:
Put a pin in this thing. The only negative `enum` like value we should be using is -1 to indicated `Invalid`, `Unassigned`, `Error`, etc. Using random negative integers to mean different things is bad design.